Dockerfile.QA

```diff
@@ -44,19 +44,34 @@ FROM ${BUILD_IMAGE} AS build
 ENV DEBIAN_FRONTEND=noninteractive
 
 # image_preprocess custom backend requires opencv
+# go client requires golang
 RUN apt-get update && apt-get install -y --no-install-recommends \
             libopencv-dev \
-            libopencv-core-dev
+            libopencv-core-dev \
+            golang-go \
+            protobuf-compiler
+
+apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
This can't be in a dockerfile.... doesn't it give you an error? And it is redundant with above.

Reply:
That was doubly added by mistake